### PR TITLE
Added four new layouts

### PR DESCRIPTION
**Pull request by OrBaruk**
Here are some new layouts, I was  not able to test them with multiple monitors.

* **columns:**: All windows uses the  full height of the screen while the horizontal space equally shared.

* **rows:** All windows  uses the full width of the screen while the vertical space is equally shared.

* **geometric progression (gp):** First window occupy half the screen, second window half of the remaining and so on until the last window occupy all the remaining space, has a vertical and horizontal version.

### CODE REVIEW

**Comment by csaunders** [comment]
Could you include some screenshots of these layouts?

**Comment by OrBaruk** [comment]
Here follows some screenshots:

**Columns**
columns


**Rows**
rows


**GP-Horizontal**
gp-horizontal


**GP-Vertical**
gp-vertical

File: layouts.lua [hunk 1]
```diff
@@ -53,4 +53,118 @@ layouts['main-horizontal'] = function(windows)
...
+    end
+    y = y + height
+  else 
+    if index ~= wincount then
+      height = height / 2
+    end
+    x = x + width
+  end
+  
+  frame.x = frame.x + x
```
Comment (csaunders) [inline layouts.lua]: looks like your indentation is a little off here.

**Comment by csaunders** [comment]
This is rad. I love the golden ratio-ness of the GP ones!

**Comment by OrBaruk** [comment]
Fixed the indentation. Also, maybe someone can find a better name for the GP layouts.

**Comment by nathankot** [comment]
@OrBaruk how about `phi-horizontal`, `phi-vertical`? as in the golden ratio haha up to you ;)

**Comment by nathankot** [comment]
@OrBaruk lets stick with the `gp-` names :) also added screenshots to the readme thanks!

**Pull request merged**